iOSDevUK/ViewComponents/RemoteImage.swift

```diff
@@ -25,5 +25,13 @@ struct RemoteImageView: View {
             .loadDiskFileSynchronously()
 //            .cacheMemoryOnly()
             .fade(duration: 0.25)
+            .aspectRatio(contentMode: .fit)
```

Comment:
dont make a change here. This view is used in multiple places where the ratio is fill